CI/build-mudlet-for-windows.sh

```diff
@@ -47,10 +47,16 @@ if [ "${MSYSTEM}" = "MSYS" ]; then
   exit 2
 elif [ "${MSYSTEM}" = "MINGW32" ]; then
   export BUILD_BITNESS="32"
-  export BUILDCOMPONENT="i686"
+  MINGW_INTERNAL_BASE_DIR="/mingw${BUILD_BITNESS}"
+  export MINGW_INTERNAL_BASE_DIR
 elif [ "${MSYSTEM}" = "MINGW64" ]; then
   export BUILD_BITNESS="64"
-  export BUILDCOMPONENT="x86_64"
+  MINGW_INTERNAL_BASE_DIR="/mingw${BUILD_BITNESS}"
+  export MINGW_INTERNAL_BASE_DIR
+elif [ "${MSYSTEM}" = "CLANG64" ]; then
```

Comment:
BTW There is a `CLANG32` environment nowadays as well - though I do not know if it has everything needed for a 32-Bit Windows Mudlet build...